e2e/stack.go

```diff
@@ -243,8 +243,9 @@ func (s *Stack) runMonomer(ctx context.Context, env *environment.Env, genesisTim
 	env.DeferErr("close tx db", txdb.Close)
 	mempooldb := dbm.NewMemDB()
 	env.DeferErr("close mempool db", mempooldb.Close)
-	ethstatedb := rawdb.NewMemoryDatabase()
-	env.DeferErr("close eth state db", ethstatedb.Close)
+	raw := rawdb.NewMemoryDatabase()
+	env.DeferErr("close raw db", raw.Close)
+	ethstatedb := state.NewDatabase(raw)
```

Comment:
following up with the last comment, Should I close the `TrieDB` here too or leave it as it is?

Reply:
Yes, go ahead and close the `TrieDB`. Thanks for pointing this case out!